pkg/utils/cluster.go

```diff
@@ -26,6 +28,14 @@ const (
 	controlPlaneNodeLabelKey = "node-role.kubernetes.io/control-plane"
 )
 
+var (
+	oneNode     = intstr.FromInt32(1)
+	defaultNpcl = &sriovnetworkv1.SriovNetworkPoolConfig{Spec: sriovnetworkv1.SriovNetworkPoolConfigSpec{
+		MaxUnavailable: &oneNode,
+		NodeSelector:   &metav1.LabelSelector{},
+		RdmaMode:       "shared"}}
+)
```

Comment:
should this one be empty by default so we don't change anything or we want the default to be shared?

Reply:
done